wasm_interface/build.gradle

```diff
@@ -25,15 +45,24 @@ jar {
     archiveAppendix = "wasm_interface"
 }
 
-
 task buildRust(type: Exec) {
-    description "Build the Rust project."
+    commandLine "echo", "gg"
+    commandLine "rustup", "target", "add", osName
+    commandLine "cargo", "build", "--release", "--target="+osName
```

Comment:
Minor comments:
1. Not sure that the 'echo gg' is needed
2. I like spaces before and after the '+'